cf-agent/verify_files_utils.c

```diff
@@ -836,7 +837,7 @@ static PromiseResult SourceSearchAndCopy(EvalContext *ctx, const char *from, cha
             }
         }
 
-        if (attr.copy.purge)    /* Purge this file */
+        if (attr.copy.purge)    /* Do not purge this file */
```

Comment:
Hmm, ok. So this means it will not be purged later on? Then it's even better if you can explain in the comment that the namecache is the reason for this.